megamek/src/megamek/client/ui/swing/ClientGUI.java

```diff
@@ -323,6 +323,10 @@ public ClientGUI(Client client, MegaMekController c) {
     public IBoardView getBoardView() {
         return bv;
     }
+    
+    public BoardView1 getBoardView1() {
```

Comment:
Probably a better approach is to add "getTileSetManager()" to the interface. 

While I think that using an interface here is an unfortunate choice (we are unlikely to swap out to different BoardView implementations), having these two methods side by side is inconsistent and will cause confusion for future developers (which one do I use?)

Reply:
OK done. (I'd honestly like to remove all that ITileManager and IBoard stuff. It gets in the way all the time)

Reply:
While it's unlikely that most of them will ever have alternate implementations, my plans for multiple-map engagements require IBoard as a common interface between single and compound maps.